opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -1167,16 +1169,37 @@ def get_tracer(
             logger.error("get_tracer called with missing module name.")
         if instrumenting_library_version is None:
             instrumenting_library_version = ""
+
+        kwargs = {
+            "message": (
+                "Call to deprecated method __init__. \(You should use "
+                "InstrumentationScope\) -- Deprecated since version 1.11.1."
+            ),
+            "category": DeprecationWarning,
+            "module": "opentelemetry.sdk.trace",
+        }
+
+        # FIXME: Remove when 3.7 is no longer supported.
+        # For some reason the lineno argument in 3.7 does not seem to work.
+        # This is still safe enough because we also filter by message and
+        # module.
+        if version_info.minor > 7:
+            kwargs["lineno"]: 1191
```

Comment:
I personally do not like line number as it seems hacky and a maintenance burden. Is it not sufficient to match with the already existing `kwargs`?

Reply:
I agree, it is very inconvenient to use line numbers. If you prefer I can remove that check, I was just being extra cautious.

Reply:
I would suggest removing this. The message, category, module is already very explicit.

Reply:
Removed

Reply:
Removed